### PR TITLE
Text Updates for QA page 12/16

### PR DESCRIPTION
**Pull request by mwernimont**
Before making a pull request
----------------------------
First . . .
- [ ] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [x] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [x] Chrome
- [x] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
Temperature QA got a text update, use also received a question

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/assets/questionsAndAnswers/questionsAndAnswers.js [hunk 1]
```diff
@@ -144,6 +144,15 @@ export default {
                     },
                     {
                         "id": "useQuestion2",
+                        "question": "What does “Concept Map” mean, and why aren’t these data available here for decision making? ",
+                        "answer": "The data presented within the IWAAs water availability series of maps demonstrate a new USGS capability for calculating near real-time estimates of water budget variables.  The process (or concept) behind the data acquisition, modeling, and delivery of water temperature or water storage data throughout the lower 48 states are orchestrated using an operational pipeline in the Cloud (See About tab for more information).  While the models used for these calculations are complex and verified (" +
+                        "<a href='https://doi.org/10.5066/P9NM8K8W' " +
```
Comment (mhines-usgs) [inline src/assets/questionsAndAnswers/questionsAndAnswers.js:149]: I think we would also want to make sure we add references in the text to the references on the bottom?

File: src/assets/questionsAndAnswers/questionsAndAnswers.js [hunk 2]
```diff
@@ -231,38 +240,47 @@ export default {
                         "id": "tempQuestion1",
                         "question": "What does this map show me?",
                         "answer": "This map displays river networks with daily water temperature represented by colors shaded " + 
-                                "from light blue to dark red. These temperatures are generated from models that consider daily " +
-                                "fluctuations in environmental variables to calculate average of stream temperature conditions for " + 
-                                "a given day. This map also contains USGS real-time temperature monitoring sites that are visible at " + 
-                                "finer zoom levels. Clicking on the symbol will open a graph of the past 7 days of temperature " + 
-                                "monitoring data along-side the modeled temperature estimates."
+                                "from light blue to dark red. These temperatures are generated from models that consider daily " +
```
Comment (mhines-usgs) [inline src/assets/questionsAndAnswers/questionsAndAnswers.js:243]: never noticed this before, but shouldn't this be dark blue?

Comment (mwernimont) [reply src/assets/questionsAndAnswers/questionsAndAnswers.js:243]: I'm just copying and pasting lol.  But can easily be done!

File: src/assets/questionsAndAnswers/questionsAndAnswers.js (hunk 1 above)
Comment (mwernimont) [reply src/assets/questionsAndAnswers/questionsAndAnswers.js:149]: That makes sense

**Review by mhines-usgs — APPROVED** [review]

**Pull request merged**